File: src/test/java/day7/CamelCardsTest.java
package day7;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class CamelCardsTest {

    @Test
    void firstStar() {
        CamelCards camelCards = new CamelCards();
        assertEquals(6440, camelCards.firstStar());
    }

    @Test
    void secondStar() {
        CamelCards camelCards = new CamelCards();
        assertEquals(5905, camelCards.secondStar());
    }
}

File: src/test/kotlin/day07/CamelCardsTest.kt
package day07

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

internal class CamelCardsTest {
    @Test
    fun firstStar() {
        assertEquals(6440, CamelCards().firstStar())
    }

    @Test
    fun secondStar() {
        assertEquals(5905, CamelCards().secondStar())
    }
}